Arm/rover_arm_system.hpp

```diff
@@ -151,19 +151,19 @@ class RoverArmSystem : public sjsu::common::RoverSystem
     heartbeat_count_++;
   }
 
-  void MoveRotunda(double angle)
+  void MoveRotunda(float angle)
   {
     rotunda_.SetSpeed(mc_data_.arm_speed);
     rotunda_.SetPosition(angle);
   }
 
-  void MoveShoulder(double angle)
+  void MoveShoulder(float angle)
   {
     shoulder_.SetSpeed(mc_data_.arm_speed);
     shoulder_.SetPosition(angle);
   }
 
-  void MoveElbow(double angle)
+  void MoveElbow(float angle)
```

Comment:
almost feel as if we could do away with passing arguments. Don't know if we talked about that earlier but just something I noticing now